### PR TITLE
Fix/remove unnecessary ui components

### PR DESCRIPTION
**Pull request by TimDaub**
TODOs:

- [x] all links to the "Exchange.js" view. Ideally remove Exchange.js
- [x] RequestFunds.js and "Request" button in App.js
- [x] In "Advanced", hide "Display currency" and "Enable advanced features"
- [x] Adjust "Code" and "About" in "Advanced" (actually make them configurable with config.js and send a PR upstream <3)
- [x] Adjust <title /> and all other meta tags/data in index.html
- [x] Cleanup eslint errors

Other things that have been done:

- Removed "message" from the "send" dialogue. Messages are anyways not stored in history at this point. They'd have to be fixed upstream before they're usuable.
- Removed the red "alpha version warning"

What hasn't been done

- There's two places where I wasn't able to replace an URL in index.html's metadata as there's no domain for Planet A yet. To fix this, I've created #14

### CODE REVIEW

**Comment by MaxStalker** [comment]
I'm starting a review process now.

File: src/components/SendToAddress.js [hunk 1]
```diff
@@ -337,7 +337,10 @@ export default class SendToAddress extends React.Component {
               </InputInfo>
             ) : null}
           </Field>
-          <Field mb={3} label={messageText}>
+          {/* For Planet A and since messages currently don't work, we simply
+              set them to display: "none".
+            */}
+          <Field style={{display: "none"}} mb={3} label={messageText}>
```
Comment (MaxStalker) [inline src/components/SendToAddress.js]: Since this is rimble - you can simply use ```display="none"``` here instead of inlinening styles

**Review by MaxStalker — CHANGES_REQUESTED** [review]
Looking good! Aside small thing with display prop :)

**Comment by MaxStalker** [comment]
I was also wondering - do we really need address field here, when we first arrive on this screen?
image

Also, I would recommend to remove "Scan QR code" button, then skip all extra steps and open camera for scanning right away, like if we press that it. Cause noone gonna bother entering address by hand anyway :)

**Comment by TimDaub** [comment]
> Also, I would recommend to remove "Scan QR code" button, then skip all extra steps and open camera for scanning right away, like if we press that it. Cause noone gonna bother entering address by hand anyway :)

That's a pretty controversial change of the UI in my opinion and I'm sure I could collect multiple points on why I believe it's problematic. Additionally, it's not within the scope of this PR. Of course, if you want to move this forward, feel free to open an issue on a repository where you see fit!

**Review by MaxStalker — APPROVED** [review]
All good :)

**Pull request merged**